js/options/colorfulAudio.js

```diff
@@ -0,0 +1,101 @@
+/* global C2D */
+window.options.colorfulAudio = {
```

Comment:
at the moment it isn't super clear what this colorfulAudio option does. the first time i used it i didn't see any effect, but later i realized that it only effects non-transparent pixels on the canvas, meaning i need to draw something first. After i made that realization, it still wasn't very clear exactly *how* the recording was effecting the image. (i figured that out later when i read the code, which i have other thoughts on. see comment below).

so i think it might be helpful to maybe include some instructions in the option itself which communicates to the user how they're supposed to use it. For example, u could explain that there needs to be something on the canvas first && that the length of the sound && volume level will change the colors of the pixels in the canvas.